### DISCUSSION TITLE
Getting consistent control between buffer and minibuffer

### DISCUSSION THREAD

**Discussion by alexarice** [meow-edit/meow#460] [Q&A] [answered]
Hi,

I've been using xah fly keys for a while but have recently been looking into using meow mode mainly due to being annoyed by not being able to use default keybindings very often.

The current main pain point for me after switching is that meow mode does not seem to function/give a consistent editing style in the minibuffer. For reference, with my previous setup, I was able to enter normal mode and navigate by going backwards and forwards words and deleting words, as well as using navigational keys (equivalent of `meow-next` and `meow-prev`) to choose an option in things like `vertico` completion lists.

Is there any way I can achieve a similar sort of behaviour in meow mode?

Thanks

**Comment by DogLooksGood** [comment]
No, there isn't. You have to use default key bindings.

**Comment by alexarice** [comment]
Could you expand on this? Is it just that by default this isn't turned on or is there a fundamental reason that meow normal state can't function in a minibuffer?

**Comment by DogLooksGood** [comment]
It's a decision made in early stage. It's probably subjective, switching modes in minibuffer doesn't save much keys for you. And using `M-f` and `M-b` isn't a bad idea. Typing in minibuffer is more like the case you do in console, it's a simple case thus modal editing is not necessary.

**Comment by alexarice** [comment]
It's not so much about saving keys, as having the same editing flow in the minibuffer. Just to make sure, trying to hack it to naively start normal state in the minibuffer won't function nicely?

**Reply by DogLooksGood** [reply, marked as answer]
It should works, there's nothing special here, except there's no point to use KEYPAD and BEACON in minibuffer.
